frontend/src/features/workspace/WorkspacePage.tsx

```diff
@@ -44,10 +44,10 @@ export const WorkspacePage = (): JSX.Element => {
     [user],
   )
   const [hasSeenAnnouncement, setHasSeenAnnouncement] =
-    useLocalStorage<boolean>(ROLLOUT_ANNOUNCEMENT_KEY)
+    useLocalStorage<boolean>(ROLLOUT_ANNOUNCEMENT_KEY, false)
 
   const isAnnouncementModalOpen = useMemo(
-    () => !isUserLoading && !hasSeenAnnouncement,
+    () => !isUserLoading && hasSeenAnnouncement === false,
```

Comment:
We should follow the same convention as @karrui mentioned - `!hasSeenAnnouncement`?

Reply:
falseness is dangerous, hasSeenAnnouncement can be `undefined`

Reply:
Would recommend having a default value but this works too 👍🏻

Reply:
the localstorage call has a default value to be set, which is why false check is good. there are 4 scenarios that could happen and all requires slightly different behaviour;

fetching from localstorage: undefined
fetched from localstorage, no key: false
fetched from localstorage, key is false: false
fetched from localstorage, key is true: true